source/winVersion.py

```diff
@@ -217,5 +217,9 @@ def isUwpOcrAvailable():
 	}
 
 
-def isFullScreenMagnificationAvailable():
+def isFullScreenMagnificationAvailable() -> bool:
+	"""
+	Technically this is always False. The Magnification API has been marked by MS as unsupported for
+	WOW64 applications such as NVDA. For our usages, support has been added since Windows 8.

```

Comment:
Please add a note here that we are "relying on our testing our specific usage of the API with each Windows version since Windows 8"